packages/wallets/src/features/cashier/modules/Transactions/components/TransactionsPendingRow/TransactionsPendingRow.tsx

```diff
@@ -34,30 +34,29 @@ const TransactionsCryptoRow: React.FC<TProps> = ({ transaction }) => {
     }, [modal, mutate, transaction.id]);
 
     const onCancelButtonClick = useCallback(() => {
-        if (isMobile)
-            modal.show(
-                <WalletActionModal
-                    actionButtonsOptions={[
-                        {
-                            onClick: modal.hide,
-                            text: "No, don't cancel",
-                        },
-                        {
-                            isPrimary: true,
-                            onClick: cancelTransaction,
-                            text: 'Yes, cancel',
-                        },
-                    ]}
-                    description='Are you sure you want to cancel this transaction?'
-                    hideCloseButton
-                    title='Cancel transaction'
-                />,
-                { defaultRootId: 'wallets_modal_root' }
-            );
-    }, [cancelTransaction, isMobile, modal]);
+        modal.show(
+            <WalletActionModal
+                actionButtonsOptions={[
+                    {
+                        onClick: modal.hide,
+                        text: "No, don't cancel",
+                    },
+                    {
+                        isPrimary: true,
+                        onClick: cancelTransaction,
+                        text: 'Yes, cancel',
+                    },
+                ]}
+                description='Are you sure you want to cancel this transaction?'
+                hideCloseButton
+                title='Cancel transaction'
+            />,
+            { defaultRootId: 'wallets_modal_root' }
+        );
+    }, [cancelTransaction, modal]);
 
-    const onMobileStatusClick = useCallback(
-        () =>
+    const onMobileStatusClick = useCallback(() => {
+        if (isMobile) {
```

Comment:
why its needed on first place? 
I would think that "onMobileStatusClick" is not visible / not called when !isMobile?

Reply:
Well, here's why it's in the function https://github.com/binary-com/deriv-app/pull/11289#discussion_r1393496005

Reply:
ah, but isn't it a completely separate part of .html/jsx which is only visible on mobiles? I might be wrong but by briefly reading the code it seemed that those checks shouldn't be needed at all to begin with...